daemon/graphdriver/devmapper/deviceset.go

```diff
@@ -388,6 +413,11 @@ func (devices *DeviceSet) deviceFileWalkFunction(path string, finfo os.FileInfo)
 		return nil
 	}
 
+	if finfo.Name() == transactionMetaFile {
+		logrus.Debugf("Skipping file %s", path)
```

Comment:
This already got merged as part of other PR. I will take care of it when I am doing a general cleanup to bring all the messages to a consistent prefix as part of separate PR.